README.md

```diff
@@ -23,12 +23,13 @@ Code from this repository powers the Q# development experience on <https://quant
 
 ## Building
 
-To build this repository there are 4 dependencies that need to be installed. These are:
+To build this repository there are dependencies that need to be installed. These are:
 
 - Python (<https://python.org>)
 - Rust (<https://www.rust-lang.org/tools/install>)
 - Node.js (<https://nodejs.org/>)
 - wasm-pack (<https://rustwasm.github.io/wasm-pack/installer/>)
+- cmake (<https://cmake.org/>) and a C compiler
```

Comment:
Could you also update `prereqs.py` to account for the new dependency?

Reply:
And perhaps a more precise link to on how to install the "C compiler" for various platforms (mac, ubuntu, windows). Will any compiler do or do we need a specific one in the toolchain?

(I know I sound like I'm nitpicking here, but it's been about 15 years since I've manually installed a C compiler 😆 .  This would definitely stop me in my tracks if I was new to contributing to this repo, or setting up a fresh environment after having been away for a while.)

Reply:
Ian and I discussed this. We'll update the docs. Windows is actually the easy one (which I think you use), as to get Rust working you must have already installed the C++ compiler (either with VS or the Build Tools - see https://learn.microsoft.com/en-us/windows/dev-environment/rust/setup ).